data/config-schema/src/main/resources/net/corda/schema/configuration/p2p.linkManager/1.0/corda.p2p.linkManager.json

```diff
@@ -94,6 +94,12 @@
       "type": "integer",
       "default": 4,
       "minimum": 1
+    },
+    "sessionRefreshThreshold": {
+      "description": "Session refresh threshold in milliseconds. The default is five days.",
```

Comment:
should we maybe have this be in seconds? Granularity of milliseconds seems a bit of an overkill here.